Tweak.xm

```diff
@@ -263,13 +264,30 @@ static void activatePiP(YTPlayerPIPController *controller) {
         MLPIPController *pip = [controller valueForKey:@"_pipController"];
         if ([controller canEnablePictureInPicture])
             [pip activatePiPController];
-        else
-            [pip deactivatePiPController];
+        // else
+        //     [pip deactivatePiPController];
```

Comment:
Why?

Reply:
> Why?

For some reason, it goes for `deactivate` way too frequently. I couldn't find any bad effect that removing those 2 lines of code create (correct me if I'm wrong), so I don't completely understand the reason behind it.